lib/api/TendermintRPCClient.js

```diff
@@ -0,0 +1,36 @@
+const { client: JaysonClient } = require('jayson/promise');
+
+class TendermintRPCClient {
+  constructor(host, port) {
+    this.client = JaysonClient.http({
+      host,
+      port,
+    });
+  }
+
+  /**
+   * Fetches transitions by Event (formerly Tag).
+   * A tag is either just a key or a key value combination.
+   * @param {string} key
+   * @param {string} [value='']
+   * @param {boolean} [prove=false]
+   * @param {string} [page='1']
+   * @param {string} [perPage='30']
+   * @return {Promise<Object[]|null>}
+   */
+  async fetchTransitionsByEvent(key, value = '', prove = false, page = '1', perPage = '30') {
```

Comment:
it's better to have a more business logic oriented method that responds with total_count